README.md

```diff
@@ -523,3 +541,47 @@ sudo sh -c "echo '127.0.0.1 pgadmin.local' >> /etc/hosts"
 
 After you have started the container open `pgadmin.local` in a web browser. The password for the `nextcloud.local` is `postgres`.
 That's it, open the following path to see the Nextcloud tables: `Server group 1 -> nextcloud.local -> Databases -> nextcloud -> Schemas -> public -> Tables`
+
+
+## Database (mysql) access
+
+```
+docker-compose run  database-mysql bash -c 'mysql -h database-mysql -u root -p${MYSQL_ROOT_PASSWORD}'
+```
```

Comment:
Unrelated change already in https://juliushaertl.github.io/nextcloud-docker-dev/services/database/